ext/libmongocrypt/extconf.rb

```diff
@@ -0,0 +1,69 @@
+require 'mkmf'
+require 'rbconfig'
+
+CMAKE = find_executable('cmake')
+abort "ERROR: CMake is required to build libmongocrypt" unless CMAKE
+
+MAKE = find_executable('gmake') || find_executable('make')
+abort "ERROR: make is required to build libmongocrypt" unless MAKE
+
+# Specify the name of the C++ source file for the extension
+extension_name = 'libmongocrypt'
+
+# Specify the directory where the CMake project is located
+cmake_dir = File.join(File.expand_path(File.dirname(__FILE__)), extension_name)
+
+# Exclude C# bindings
+cmake_lists_file = File.join(cmake_dir, 'CMakeLists.txt')
+text = File.read(cmake_lists_file)
+File.open(cmake_lists_file, "w") do |file|
+  file << text.gsub(/^add_subdirectory \(bindings\/cs\)$/, "# add_subdirectory (bindings/cs)")
+end
+
+# Specify the build directory for CMake
+build_dir = File.join(cmake_dir, 'build')
+
+# Run CMake to generate the build files
+Dir.mkdir(build_dir) unless File.exist?(build_dir)
+cmake_opts = %w(
+  -DDISABLE_NATIVE_CRYPTO=1
+  -DMONGOCRYPT_CRYPTO=none
+  -DMONGOCRYPT_ENABLE_CRYPTO=0
+  -DBUILD_TESTING=0
+  -DBUILD_VERSION=1.7.4
```

Comment:
Could the `BUILD_VERSION` be set from `LibmongocryptHelper::VERSION`?

Reply:
Good point, thank you! Fixed.